packages/grpcts/src/Client.ts

```diff
@@ -32,18 +39,38 @@ export class Client {
     metadata?: Metadata
   ): { call: grpc.ClientUnaryCall; res: Promise<ResponseType> } {
     let call: grpc.ClientUnaryCall | undefined;
+    const latency = this.latencyTimer.start();
     const res = new Promise<ResponseType>((resolve, reject) => {
       call = this.client[methodName](
         req,
         this.metadata(metadata),
         (err: grpc.ServiceError, res: ResponseType) => {
-          err ? reject(this.convertError(err)) : resolve(res);
+          if (err) {
+            return reject(this.convertError(err));
+          }
+
+          this.log(methodName, req, latency);
```

Comment:
We log only latency of successful responses?

Reply:
yes. will adjust for all requests